etc/resources/anvil.json

```diff
@@ -0,0 +1,25 @@
+{
+  "types": [],
+  "domains": [
+    {
+      "domain_name": "AnvilZKS",
```

Comment:
Shouldn't it be `AnvilZksync` / `anvil_zksync`? ZKS feels super odd.

Reply:
Agree